README.md

```diff
@@ -1,19 +1,18 @@
 # Vault Secrets Reloader
 
 Vault Secrets Reloader can periodically check if a secret that is used in watched workloads has a new version in
-Hashicorp Vault, and if so, automatically “reloads” them by incrementing an annotation value, initiating a rollout for
-the workload’s pods. This controller is essentially a complementary to `vault-secrets-webhook`, relying on it for
+Hashicorp Vault, and if so, automatically “reloads” them by incrementing an annotation value, then initiating a rollout for
```

Comment:
I think the original is more correct here, as incrementing the annotation value is the act that triggers Kubernetes for a new rollout, not a following step done by the Reloader.